gateway/api/services/file_storage.py

```diff
@@ -151,13 +151,19 @@ def get_file(self, file_name: str) -> Optional[Tuple[FileWrapper, str, int]]:
             )
             return None
 
-        with open(path_to_file, "rb") as file_object:
-            file_wrapper = FileWrapper(file_object)
+        file_type = mimetypes.guess_type(path_to_file)[0]
+        file_size = os.path.getsize(path_to_file)
 
-            file_type = mimetypes.guess_type(path_to_file)[0]
-            file_size = os.path.getsize(path_to_file)
+        # We can not use context manager here. Django close the file automatically:
+        # https://docs.djangoproject.com/en/5.1/ref/request-response/#fileresponse-objects
+        file_wrapper = FileWrapper(
+            open(path_to_file, "rb")  # pylint: disable=consider-using-with
```

Comment:
I don't know why the `gateway` tests were working before but it seems that to transfer a file we don't need to setup a context manager. It's being specified in the documentation that I just pasted there.